README/ReleaseNotes/v610/index.md

```diff
@@ -248,6 +248,7 @@ The following interfaces have been removed, after deprecation in v6.08.
 ## Parallelism and PROOF
 - Add ROOT::GetImplicitMTPoolSize function to get the size of the pool used to enable implicit multi threading
 - Add the TThreadExecutor::Foreach method for parallelising functions featuring void return type
+- Add TBufferMerger and TBufferMergerFile to facilitate writing in parallel to a single output file
```

Comment:
Good. I would keep the first 2 (out 3) paragraphs.  I would also add a link and an extract to the tutorials for example:
```
   // Create the TBufferMerger
   TBufferMerger merger("mp103_fillNtuple.root");

   // Define what each worker will do
   auto work_function = [&]() {
      auto f = merger.GetFile();
      TNtuple ntrand("ntrand", "Random Numbers", "r");
      fill(ntrand, nEventsPerWorker);
      ntrand.Write();
      f->Write();
   };
```